app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+end
```

Comment:
Is there a reason why you have both User and admin models? :thinking:

Reply:
**Thanks for code review.**
We were using User to test the project but we forgot to deleted it. So now it's gone 
Thank you 💻  🚀